### ISSUE TITLE
Any chance of React 16 support?

### ISSUE THREAD

**Issue by ghost** [RRutsche/react-parallax#35]
```
npm WARN react-parallax@1.4.4 requires a peer of react@^0.14.9 || ^15.3.0-0 but none was installed.
npm WARN react-parallax@1.4.4 requires a peer of react-dom@^0.14.9 || ^15.3.0-0 but none was installed.
```

Get this warning when transitioning to react 16. It still works fine from what I can see.

**Comment by rrutsche** [comment]
I will have a look

**Comment by ghost** [comment]
Many thanks :) I think it just needs an addition to the peerDependencies

**Comment by rrutsche** [comment]
I just published version 1.5.0 on npm with updated peer dependencies.

regards

**Comment by ghost** [comment]
Thanks @RRutsche 👍 👍
